suite-native/app/src/App.tsx

```diff
@@ -34,6 +34,11 @@ const APP_STARTED_TIMESTAMP = Date.now();
 // Keep the splash screen visible while we fetch resources
 SplashScreen.preventAutoHideAsync();
 
+// Global configuration of NetInfo for network status monitoring.
```

Comment:
Wouldn't be better place for this component in file with function definition?

Reply:
You mean for the comment? I was also thinking about that, but I believe it's better to see here why it is being called and what is it about. I don't expect us to expand this in any major way in the future. But both places seem fine to me. @vytick , we need one more vote 😉

Reply:
I'll merge as it is, anyone feel free to open PR to enhance the comment 🙏